mappings/net/minecraft/entity/boss/dragon/EnderDragonFight.mapping

```diff
@@ -1 +1,10 @@
 CLASS brj net/minecraft/entity/boss/dragon/EnderDragonFight
+	FIELD a LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD e gateways Ljava/util/List;
+	FIELD k dragonKilled Z
+	FIELD l previouslyKilled Z
+	FIELD m dragonUUID Ljava/util/UUID;
```

Comment:
dragonUUID or dragonUuid?

Reply:
`dragonUUID`, matches the NBT key.

Reply:
I thought we agreed not to capitalize UUID (or NBT, or...) in field/method names, though. Which do you think is more important (genuine question) - matching NBT keys or keeping to an unified naming standard?

Reply:
I didn't realise that was agreed upon - `dragonUuid` then.